### DISCUSSION TITLE
Microsoft Defender - Severe Reaction

### DISCUSSION THREAD

**Discussion by SonnyDL** [SubtitleEdit/subtitleedit#5464] [General]
Microsoft Defender is very sure this contains a dangerous trojan

Detected: Trojan:MSIL/AgentTesla.SMK!MTB
Details: This program is dangerous and executes commands from an attacker.

**Comment by niksedk** [comment]
What version of SE are you using?
SE has often been flagged - from SE 3.6.3 SE will no longer use an exe packer (ILRepack) in order to make this a little better.
Do report "false positive" to your anti-virus provider, thx.

Read more here: https://github.com/SubtitleEdit/subtitleedit/issues/5280

**Reply by SonnyDL** [reply]
This happened for the 3.6.2, both the SE and the portable. A couple of scanners in VirusTotal also confirmed the trojan.

**Comment by OmrSi** [comment]
@niksedk I tried the beta version on another laptop with Windows 11 and the defender just kept deleting it, I had to turn it off.

**Comment by niksedk** [comment]
I get one flag on virus total: https://www.virustotal.com/gui/file/db5a2d01571a3a1e66895b2ae52723e4bdfc0a87769c3e14d66db692a9fac5da
At 3.6.2 release it was 28...

But as always, please report "false positive" to your anti-virus provider, thx.
